test/models/imports/episode_timings_import_test.rb

```diff
@@ -0,0 +1,127 @@
+require "test_helper"
+
+describe EpisodeTimingsImport do
+  let(:import) { EpisodeTimingsImport.new }
+
+  describe ".parse_timings" do
+    it "parses timings" do
+      assert_equal [1.23], EpisodeTimingsImport.parse_timings("1.23")
+      assert_equal [1.23, 4.56], EpisodeTimingsImport.parse_timings(" 1.23 , 4.56")
+      assert_equal [1.23, 4.56, 6], EpisodeTimingsImport.parse_timings(" 1.23 , 4.56,6")
```

Comment:
what makes the spacing behave this way? (the leading whitespace, one comma has whitespace on both sides while the other doesn't)

Reply:
This test is just proving that the code ignores/strips whitespace.  It's a bit of a nonsensical string here - as if the user had done a bad job typing into excel.